app/views/users/admin/userAdmin.scala.html

```diff
@@ -0,0 +1,129 @@
+@import db.ModelService
+@import db.impl.access.{UserBase, ProjectBase, OrganizationBase}
+@import models.user.User
+@import models.user.role.{OrganizationRole, ProjectRole, RoleModel}
+@import ore.permission.role.{RoleTypes, GlobalRole}
+@import ore.{OreConfig, OreEnv}
+
+@(user: User)(implicit messages: Messages, request: Request[_], service: ModelService,
+        config: OreConfig, users: UserBase, projects: ProjectBase, organizations: OrganizationBase)
+
+@isOrg = @{user.isOrganization}
+
+@orgRoles = @{RoleTypes.values.filter(r => r.roleClass == classOf[OrganizationRole] && r.roleId != 64)}
+@projectRoles = @{RoleTypes.values.filter(_.roleClass == classOf[ProjectRole])}
+@globalRoles = @{RoleTypes.values.filter(_.roleClass == classOf[GlobalRole])}
+
+
+@panel(title: String, size: Int = 9)(content: Html) = {
+    <div class="col-md-@size">
+        <div class="panel panel-default">
+            <div class="panel-heading">
+                <h4 class="panel-title">@title</h4>
+            </div>
+            @content
+        </div>
+    </div>
+}
+
+@roleTable[M <: RoleModel](keyName: String, roles: Set[M], roleTypes: Set[RoleTypes.Value])(showKey: M => Html) = {
+    <table class="table">
+        <tr><th>@keyName</th><th>Role</th><th>Accepted</th></tr>
+        @for(role <- roles) {
+        <tr>
+            <td>@showKey(role)</td>
+            <td><select>
+                @for(roleType <- roleTypes) {
+                    <option @if(role.roleType == roleType) {selected}>@roleType.title</option>
+                }
+            </select></td>
+            <td><input type="checkbox" @if(role.isAccepted) {checked}></td>
+            <td><a href="#"><i class="fa fa-trash"></i></a></td>
+        </tr>
+        }
+        <tr>
+            <td colspan="3">
+                <button data-toggle="modal" data-target="#new-page" title="Add" class="btn yellow btn-sm">
+                        <i class="fa fa-plus"></i>
+                </button>
+            </td>
+        </tr>
+    </table>
+}
+
+@bootstrap.layout((if(isOrg) "Organization" else "User") + " Admin - " + user.username) {
```

Comment:
I don't think this is giving me the information I want. Seeing `User Admin - simon` confuses me. Is `simon` an Admin? What does this page do? `Edit User - simon` would be clearer.